tests/yam/agama/agama.pm

```diff
@@ -27,6 +27,10 @@ use Utils::Architectures qw(is_s390x is_ppc64le);
 use Utils::Backends qw(is_svirt);
 use power_action_utils 'power_action';
 
+sub is_headless {
```

Comment:
`is_headless_installation` would be more clear.

Reply:
changed.